backend/app/services/knowledge/knowledge_service.py

```
@@ -3192,3 +3194,223 @@ def migrate_knowledge_base_to_group(
+                from app.services.rag.runtime_resolver import RagRuntimeResolver
+
+                delete_runtime_spec = RagRuntimeResolver().build_delete_runtime_spec(
+                    db=db,
+                    knowledge_base_id=source_kb_id,
+                    document_ref=str(doc.id),
+                    index_owner_user_id=kb_info.index_owner_user_id,
+                )
+                _run_async_in_new_loop(
+                    rag_gateway.delete_document_index(delete_runtime_spec, db=db)
+                )
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)